.github/workflows/ci.yml

```diff
@@ -432,6 +432,10 @@ jobs:
           java-package: jdk
           architecture: x64
 
+      - shell: bash
+        if: "matrix.suite == 'integration_tests'"
+        run: pip install lmdb
+
```

Comment:
@kquick (or anyone else with CI knowledge) – any idea why this `pip install` might not be carrying over to the run of `test_solver_cache/test.sh`? I had to [add the same line to `test.sh`](https://github.com/GaloisInc/saw-script/blob/6a5ec6725ce0b53f41864694b266779f3a3811bb/intTests/test_solver_cache/test.sh#L7) in order to get the test to pass on CI.